vhdbuilder/packer/windows-image.env

```diff
@@ -0,0 +1,5 @@
+# Windows base images for building VHD. To install Windows patches, please refer to configure-windows-vhd.ps1
+WINDOWS_2019_BASE_IMAGE_SKU=2019-Datacenter-Core-smalldisk
+WINDOWS_2019_BASE_IMAGE_VERSION=17763.1339.2007101755
```

Comment:
Should we upgrade it to a newer version?

Reply:
If we use 9x patch, this have to be upgraded to the August version. The patch and base image work in pairs. I can add an explanation of patch base image combination, but prefer to add new version when we patch 9B.

Reply:
reference: https://github.com/Azure/aks-engine/pull/3692

Reply:
I think that we can update the base image version before a newer patch is coming. I think that we should always try to update it to the latest version. It will also save the build time if we need to release more than one VHD in one month.

Reply:
Updated both 2019 and 2004 base image.

➜  ~ az vm image show --urn MicrosoftWindowsServer:WindowsServer:2019-Datacenter-Core-with-Containers-smalldisk:latest

{
  "automaticOsUpgradeProperties": {
    "automaticOsUpgradeSupported": false
  },
  "dataDiskImages": [],
  "disallowed": {
    "vmDiskType": "None"
  },
  "hyperVgeneration": "V1",
  "id": "/Subscriptions/c1089427-83d3-4286-9f35-5af546a6eb67/Providers/Microsoft.Compute/Locations/westus/Publishers/MicrosoftWindowsServer/ArtifactTypes/VMImage/Offers/WindowsServer/Skus/2019-Datacenter-Core-with-Containers-smalldisk/Versions/17763.1397.2008070242",
  "location": "westus",
  "name": **"17763.1397.2008070242"**,
  "osDiskImage": {
    "operatingSystem": "Windows",
    "sizeInBytes": 32214352384,
    "sizeInGb": 31
  },
  "plan": null,
  "tags": null
}
➜  ~  az vm image show --urn MicrosoftWindowsServer:WindowsServer:datacenter-core-2004-with-containers-smalldisk:latest
{
  "automaticOsUpgradeProperties": {
    "automaticOsUpgradeSupported": false
  },
  "dataDiskImages": [],
  "disallowed": {
    "vmDiskType": "None"
  },
  "hyperVgeneration": "V1",
  "id": "/Subscriptions/c1089427-83d3-4286-9f35-5af546a6eb67/Providers/Microsoft.Compute/Locations/westus/Publishers/MicrosoftWindowsServer/ArtifactTypes/VMImage/Offers/WindowsServer/Skus/datacenter-core-2004-with-containers-smalldisk/Versions/19041.450.2008080726",
  "location": "westus",
  "name": **"19041.450.2008080726"**_,
  "osDiskImage": {
    "operatingSystem": "Windows",
    "sizeInBytes": 32214352384,
    "sizeInGb": 31
  },
  "plan": null,
  "tags": null
}